packages/calculator/test/Main-test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import Main from '../src/main';
+
+// describe what we are testing
+describe('Open Calculator Button', () => {
```

Comment:
can you use `jest`, the plan is to use that going forwards along with enzyme and react-test-renderer.